app/client/components/footer.tsx

```diff
@@ -5,40 +5,40 @@ import { css } from "../styles/emotion";
 const Footer = () => (
   <footer>
     <div
-      className={css({
+      css={{
         backgroundColor: palette.neutral.header,
         color: palette.neutral["1"],
         height: "42px",
         "text-align": "right"
-      })}
+      }}
     >
       <div
-        className={css({
+        css={{
           "max-width": "71.25rem",
           margin: "auto",
           "line-height": "42px",
           "padding-right": "2rem"
-        })}
+        }}
       >
-        <a href={"#top"} className={css({ color: palette.neutral["1"] })}>
+        <a href={"#top"} css={{ color: palette.neutral["1"] }}>
           back to top
         </a>
       </div>
     </div>
     <div>
       <div
-        className={css({
+        css={{
           backgroundColor: "#333",
```

Comment:
Please can we pick a consistent code style for these? e.g. all camel, or all quoted property keys.

Reply:
We've gone for camelCase.